src/app/hearing-part/components/hearings-table/hearings-table.component.ts

```diff
@@ -78,11 +80,16 @@ export class HearingsTableComponent implements AfterViewChecked {
     }
 
     hasNotes(hearing: HearingForListingWithNotes): boolean {
-        return hearing.notes.length > 0;
+        return hearing.notes.filter(note => {
+            return note.type === NoteType.OTHER_NOTE ||
```

Comment:
We have a helper method `isOfTypeOtherOrListing`. Maybe we should add something similar here?

This probably has some business meaning, such as: "isNoteAllowedTo..."

Reply:
changed